src/components/CurriculumComponents/CurriculumDownloadTab/index.tsx

```diff
@@ -60,7 +60,7 @@ function ScrollIntoViewWhenVisisble({
   return <div ref={ref}>{children}</div>;
 }
 
-export function createCurriculumDownloadsQuery(
+export function createCurriculumDownloadsUrl(
```

Comment:
This function should be moved into a new file `./src/utils/curriculum/url.ts` we shouldn't be importing functions between components